x/ccv/parent/keeper/proposal_test.go

```diff
@@ -34,7 +35,8 @@ func (suite *KeeperTestSuite) TestMakeChildGenesis() {
 	actualGenesis.ParentConsensusState.NextValidatorsHash = []byte{}
 	expectedGenesis.ParentConsensusState.NextValidatorsHash = []byte{}
 
-	actualGenesis.InitialValSet[0].PubKey = crypto.PublicKey{}
+	// set valset to one empty validator because SetupTest() creates 4 validators per chain
+	actualGenesis.InitialValSet = []abci.ValidatorUpdate{{PubKey: crypto.PublicKey{}, Power: actualGenesis.InitialValSet[0].Power}}
```

Comment:
@jtremback this test was failing after the IBC-GO branch valset update. Lmk if this fix is ok with you.